docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -32,7 +110,9 @@ This will install dlt with **databricks** extra which contains Databricks Python
 
 This should have your connection parameters and your personal access token.
 
-It should now look like:
+You will find your server hostname and HTTP path in the your cluster settings -> Advanced Options -> JDBC/ODBC.
```

Comment:
If it's indeed using a SQL warehouse and not a Spark cluster, then this should be something like: "go to your SQL warehouse -> Connection details".

Reply:
Yes! The cluster was working, but overkill in this case.  

Is the default warehouse created automatically? It was already there in the account when I took over.

Reply:
It is indeed created automatically.

![image](https://github.com/dlt-hub/dlt/assets/47451109/af2aec9e-b0f4-4ef2-9553-6160495ee339)
https://learn.microsoft.com/en-us/azure/databricks/compute/sql-warehouse/create-sql-warehouse#what-is-a-sql-warehouse